packages/slo-stat-bot/src/slo-lint.ts

```diff
@@ -145,18 +147,57 @@ async function getFileShaContents(
  */
 export const lint = async function lint(
   schema: JSON,
-  sloData: JSON
+  sloData: SLORules[]
 ): Promise<ValidationResults> {
   const ajv = new Ajv();
   const validate = ajv.compile(schema);
-  const isValid = await validate(sloData);
+  let isValid = await validate(sloData);
+
+  if (validate.errors) {
+    return {
+      isValid: isValid,
+      errors: validate.errors,
+    } as ValidationResults;
+  }
+
+  //Check for contradicting SLOs
+  let errors = null;
+  for (const slo of sloData) {
+    isValid = !(await sloContradicts(slo));
+
+    if (!isValid) {
+      const sloString = JSON.stringify(slo);
```

Comment:
This is going to provide some ugly output usually.  To get prettier formatting for display in a call stack, I use `JSON.stringify(slo, null, 2);`.  Try this out in the node repl!

Reply:
I tried doing that but the output prints with '\n' in the message. I think this might be caused by calling `JSON.stringify(slo, null, 4)`  in the `createCheck` method